### PR TITLE
[amp-sidebar 1.0] [Toolbar] Added Tests for Current Toolbar functionality

### PR DESCRIPTION
**Pull request by torch2424**
Relates to #9985

Thais creates some basic unit tests for the added functionality of toolbar. Except for reaming features and styling.

### CODE REVIEW

**Comment by camelburrito** [comment]
create new file for testing toolbar.js

File: extensions/amp-sidebar/1.0/test/test-amp-sidebar-toolbar.js [hunk 1]
```diff
@@ -0,0 +1,243 @@
+
+/**
```
Comment (camelburrito) [inline extensions/amp-sidebar/1.0/test/test-amp-sidebar-toolbar.js]: rename to test-toolbar.js

File: extensions/amp-sidebar/1.0/test/test-amp-sidebar-toolbar.js [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+ import {adopt} from '../../../../src/runtime';
+ import {createIframePromise} from '../../../../testing/iframe';
+ import {timerFor} from '../../../../src/services';
+ import {toggleExperiment} from '../../../../src/experiments';
+ import * as sinon from 'sinon';
+ import '../amp-sidebar';
```
Comment (camelburrito) [inline extensions/amp-sidebar/1.0/test/test-amp-sidebar-toolbar.js]: shouldn't this import toolbar?

Comment (torch2424) [inline extensions/amp-sidebar/1.0/test/test-amp-sidebar-toolbar.js]: Toolbar requires and amp-sidebar element to be instantiated. So we need to create one for toolbar to work

Comment (camelburrito) [inline extensions/amp-sidebar/1.0/test/test-amp-sidebar-toolbar.js]: Stub one or more dummy amp-sidebar element(s), and test that is ideally how unit tests should work.

**Pull request merged**